shoryuken.gemspec

```diff
@@ -17,13 +17,13 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.6"
+  spec.add_development_dependency "bundler", '~> 1.10.6'
```

Comment:
Yup, it is. I might move Bundler back. There are test failures but I PR'd for WIP, in case you were actively working on the update.